Header header
string warehouse_name
string warehouse_id
int8 no_of_robots
int8 no_of_storage_units
int8 no_of_items
